recipes/controller.rb

```diff
@@ -26,7 +26,7 @@
 include_recipe 'memcached'
 include_recipe 'osl-openstack::identity'
 include_recipe 'osl-openstack::image'
-include_recipe 'osl-openstack::network'
+include_recipe 'osl-openstack::network' unless node['osl-openstack']['network_node']
```

Comment:
If the above `network_node` attribute signifies that there is a *seperate* network node, then `unless` makes sense here.